input/witnessgen.go

```diff
@@ -114,11 +114,6 @@ const (
 	// output that sends to a nested P2SH script that pays to a key solely
 	// under our control. The witness generated needs to include the
 	NestedWitnessKeyHash StandardWitnessType = 11
-
-	// CommitSpendNoDelayTweakless is similar to the CommitSpendNoDelay
-	// type, but it omits the tweak that randomizes the key we need to
-	// spend with a channel peer supplied set of randomness.
-	CommitSpendNoDelayTweakless StandardWitnessType = 12
```

Comment:
Where are they persisted exactly?

What we can do is just define it there (or change the name here to indicate it should not be used anymore), and convert it to the common type when reading it from disk.

Reply:
breacharbiter and nursery for sure, probably a few other places.

> What we can do is just define it there (or change the name here to indicate it should not be used anymore), and convert it to the common type when reading it from disk.

would need to see in how many places the serialization logic lives, but that's definitely one path. or we just leave the switch statements in place. since the tweakless-ness is now signaled through the sign descriptor, it might not be so bad

Reply:
Had the same thought when I saw this, IIRC they're also persisted in some tower logic as well. Even independent of that, it's a useful witness type, as I may want to pass the very same sign descriptor for the two commitment types, an example being in the future if we opt to allow _existing_ channels to switch over to this new witness type.